apps/Pac-Man/uninstall

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+sudo rm -f "/usr/local/bin/pacman"
+sudo rm -rf "/usr/local/share/pacman"
+sudo rm -f "/usr/local/share/applications/pacman.desktop"
+sudo rm -rf "/usr/local/share/doc/pacman"
+
+sudo update-desktop-database -q
+
+purge_packages || error "Dependencies failed to uninstall"
```

Comment:
```suggestion
purge_packages || exit 1
```

Reply:
is there a reason why you want me to just exit and error without telling the user what went wrong?
(this goes for all your suggestions)

Reply:
This is suggested by Botspot: https://github.com/Botspot/pi-apps/pull/1139#issuecomment-969705180